src/components/events/events.tsx

```diff
@@ -38,25 +39,44 @@ const Events: FunctionComponent = () => {
   const columns = useMemo(() => Columns, []);
 
   const [data, setData] = useState([]);
+  const [countRows, setCountRows] = useState<number>(0);
 
-  async function getData(searchParam?: string) {
+  const [page, setPage] = useState<number>(0);
+  const [itemsPerPage, setItemsPerPage] = useState<number>(rowsPerPageOptions[0]);
+
+  async function getData(page: number, itemsPerPage: number, searchParam?: string) {
     await eventsApi
-      .getEvents(searchParam)
+      .getEvents(page, itemsPerPage, searchParam)
       .then(response => {
         setData(response.content);
+
+        return response;
+      })
+      .then(response => {
+        setCountRows(response.totalElements);
+        return response;
       })
```

Comment:
can we combine these two then?
`.then(response => {
        setData(response.content);
        setCountRows(response.totalElements)
        return response;
      })`

Reply:
Yes, I'll do it